bubble-wrap-style.yaml

```diff
@@ -2429,42 +2300,33 @@ layers:
                         #color: green
                         visible: global.sdk_road_shields
 
-#            width_1char:
-#                filter: |
-#                    function() {
-#                        if( feature.shield_text ) {
-#                            return (feature.shield_text.length === 1)
-#                        } else {
-#                            return (feature.ref.length === 1)
-#                        }
-#                    }
-#                draw:
-#                   mapzen_icon_library:
-#                       size: [[6,[14px,14px]],[13,[16px,16px]],[15,[18px,18px]]]
-#            width_2char:
-#                filter: |
-#                    function() {
-#                        if( feature.shield_text ) {
-#                            return (feature.shield_text.length === 2)
-#                        } else {
-#                            return (feature.ref.length === 2)
-#                        }
-#                    }
-#                draw:
-#                   mapzen_icon_library:
-#                       size: [[6,[18px,14px]],[13,[20px,15px]],[15,[24px,18px]]]
-#            width_3char:
-#                filter: |
-#                    function() {
-#                        if( feature.shield_text ) {
-#                            return (feature.shield_text.length === 3)
-#                        } else {
-#                            return (feature.ref.length === 3)
-#                        }
-#                    }
-#                draw:
-#                   mapzen_icon_library:
-#                       size: [[6,[25px,14px]],[13,[27px,15px]],[15,[33px,18px]]]
+            width_1char:
```

Comment:
Not sure we need this section / not sure what it does (it was commented out before -- and the new version only filters, doesn't draw).